modules/tiff/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@loaders.gl/tiff",
+  "version": "3.0.0-alpha.18",
+  "description": "Framework-independent loaders for tiff and geotiff",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/visgl/loaders.gl"
+  },
+  "keywords": [
+    "webgl",
+    "loader",
+    "3d",
+    "texture",
+    "tiff",
+    "geotiff"
+  ],
+  "types": "src/index.d.ts",
+  "main": "dist/es5/index.js",
+  "module": "dist/esm/index.js",
+  "sideEffects": false,
+  "files": [
+    "src",
+    "dist",
+    "bin",
+    "README.md"
+  ],
+  "scripts": {
+    "pre-build": "npm run build-bundle",
+    "build-bundle": "webpack --display=minimal --config ../../scripts/webpack/bundle.js"
+  },
+  "dependencies": {
+    "@loaders.gl/core": "3.0.0-alpha.18",
+    "@loaders.gl/images": "3.0.0-alpha.18",
+    "@loaders.gl/loader-utils": "3.0.0-alpha.18",
+    "@loaders.gl/worker-utils": "3.0.0-alpha.18",
+    "fast-deep-equal": "^3.1.3",
+    "fast-xml-parser": "^3.16.0",
+    "geotiff": "ilan-gold/geotiff.js#ilan-gold/viv_094",
```

Comment:
@ilan-gold is your geotiff fork still required?

Reply:
If you want the abort signals and reliable LZW (de)compression, then yes.  I think those are the only two differences.

Reply:
I am trying to get the first issue resolved.  The second one is a bit hairier because it uses @manzt 's compiled-from-rust lzw decoder as well as a small change to the decompressor function

Reply:
Primary issue here is that the decoders for geotiff.js are baked into the library. In Zarr.js we a global registry which allows you to swap in/out different codec implementations, as well as allow for dynamic loading of compression modules. In contrast, geotiff JS `getDecoder` function wraps a _complete_ set of decoders and aren't tree-shakeable. https://github.com/geotiffjs/geotiff.js/issues/172#issuecomment-689651449